<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NerdPack ToolBox</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/MyFiles/Resources/icon.ico</normaloff>:/MyFiles/Resources/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>375</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="core_tab">
     <property name="accessibleName">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Core</string>
     </attribute>
     <widget class="QLabel" name="bg_grp">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>!!!BACKGROUND!!</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="core_grpbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>211</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>NerdPack's Core Settings</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>30</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Release:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Release</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Beta</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Use Protected Module</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>111</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Core Information</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Forks</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Stars</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Last Updated:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="forks_strg">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="starts_strg">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lastupdated_strg">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>0</y>
        <width>201</width>
        <height>121</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Launch World of Warcraft</string>
      </property>
      <widget class="QPushButton" name="wow86_bt">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>World of Warcraft (x86)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="wow64_bt">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>World of Warcraft (x64)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="wow_mac_bt">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>World of Warcraft (Mac)</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>751</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>World of Warcraft Location:</string>
      </property>
      <widget class="QLineEdit" name="wow_loc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>651</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="browse_bt">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>20</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="toolbox_grp">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>ToolBox Info</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Version</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>67</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>OS</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ver_strg">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="os_strg">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>113</width>
         <height>16</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="cr_tab">
     <attribute name="title">
      <string>Combat Routines</string>
     </attribute>
     <widget class="QTableWidget" name="cr_tb">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>341</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stars</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Last Update</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="modules_tab">
     <attribute name="title">
      <string>Modules</string>
     </attribute>
     <widget class="QTableWidget" name="mod_tb">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>341</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stars</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Last Update</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="stg_tab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="log_tab">
     <attribute name="title">
      <string>Log</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>372</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_bt">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>371</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/MyFiles/Resources/refresh.png</normaloff>:/MyFiles/Resources/refresh.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="install_bt">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>371</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Install/Update</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
